### ISSUE TITLE
False positive on YouTube - dynamic_categories (request scan)

### ISSUE THREAD

**Issue by ra-at-diladele-com** [diladele/websafety-issues#687]
Request processing on YouTube search page gives the following false positive
```
https://www.youtube.com/results?search_query=test
```
```
nudity_pornography nudity_pornography:90:+|query|,
```

We may of course disable scanning of the request but why only query word detected as nudity?

**Comment by ra-at-diladele-com** [comment]
The text that is being scanned is
```
www.youtube.com www.youtube.com resultssearch query test
```

**Comment by ra-at-diladele-com** [comment]
Ok seems to be fixed in latest 6.2
